cluster/install/fluo-cluster/bin/impl/init.sh

```diff
@@ -17,17 +17,9 @@ SSH_OPTS=(-tt -o 'StrictHostKeyChecking no' -A)
 
 $BIN_DIR/fluo-cluster kill &> /dev/null
 
-for host in `cat $CONF_DIR/hosts/all_hosts`; do
-  ssh "${SSH_OPTS[@]}" $CLUSTER_USERNAME@$host rm -rf /media/ephemeral*/zoo*  /media/ephemeral*/hadoop* /media/ephemeral*/yarn*
-done
+pssh -i -h $CONF_DIR/hosts/all_hosts "rm -rf /media/ephemeral*/zoo*  /media/ephemeral*/hadoop* /media/ephemeral*/yarn*"
 
-while read line; do
-  IFS=' ' read -ra ARR <<< "$line"
-  HOST=${ARR[0]}
-  SERVICES=${ARR[@]:1}
-  echo "`hostname`: Installing services on $HOST: $SERVICES"
-  ssh "${SSH_OPTS[@]}" $CLUSTER_USERNAME@$HOST $BIN_DIR/fluo-cluster install $SERVICES < /dev/null
-done < $CONF_DIR/hosts/hosts_with_services
+pssh -x "-tt -o 'StrictHostKeyChecking no'" -i -h $CONF_DIR/hosts/all_hosts "$BIN_DIR/fluo-cluster install --use-config"
```

Comment:
I looked over the scripts and added it to one more spot in  b8e601ee298

There might be one or two more locations.  However, those locations were tricky but each SSH command is being passed different information.  Also, the speed up may be negligible as the commands are not doing much.